cluster/helm/splice-validator/tests/validator_test.yaml

```diff
@@ -126,3 +125,84 @@ tests:
           value: ledger-api-user
       - exists:
           path: spec.template.spec.containers[?(@.name=='validator-app')].env[?(@.name=='ADDITIONAL_CONFIG_DISABLE_AUTH')].value
+
+  - it: "configures scan-client for trust-single mode"
```

Comment:
This all looks good. 
What do you think of adding one more step to the scanClient, to improve defaults? 

We can default to scanType: bft if no other type is set but the scanClient block is present. 
And in that scenario we can either accept scanAddress or seedUrls as seeds. 

This way client would be able to configure it simply as:

```
scanClient:
  scanAddress: address
```

or 

```
scanClient:
  seedUrls:
    - address 
```